modules/deployment/src/main/java/org/eclipse/xpanse/modules/deployment/ServiceDetailsViewManager.java

```diff
@@ -199,7 +213,94 @@ public List<DeployedService> listDeployedServicesOfIsv(Category category, Csp cs
         String namespace = userServiceHelper.getCurrentUserManageNamespace();
         query.setNamespace(namespace);
         List<ServiceDeploymentEntity> deployServices = serviceDeploymentStorage.listServices(query);
-        return deployServices.stream().map(EntityTransUtils::convertToDeployedService).toList();
+        return setServiceConfigurationForDeployedService(deployServices);
+    }
+
+    private void setServiceConfigurationDetailsForDeployedService(DeployedService details,
+            ServiceDeploymentEntity serviceDeployment) {
```

Comment:
Do we really need the serviceDeployment argument? the serviceTemplateId is also available in the DeployedService object.